src/structures/Message.js

```diff
@@ -572,7 +572,7 @@ class Message extends Base {
     }
     return (
       this.author.id === this.client.user.id ||
-      this.channel.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_MESSAGES, false)
+      this.channel.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_MESSAGES)
```

Comment:
Please revert this, this override is required due to something consider not a bug by discord (see https://github.com/discordjs/discord.js/pull/2756)